README.md

```diff
@@ -31,9 +31,12 @@ docker run -d \
   -p 30120:30120 \
   -p 30120:30120/udp \
   -v /volumes/fivem:/config \
+  -t \
```

Comment:
nit: These are often used together like `  -ti \`
Is the interactive/tty actually required for this?

Reply:
It's required to prevent SIGSEGV exeption